app/helpers/amendments_helper.rb

```diff
@@ -2,7 +2,9 @@
 
 module AmendmentsHelper
   def collection_title(collection)
-    content_tag(:b, 'Elige ' + collection.first.class.model_name.human.downcase) + ':<br>'.html_safe if collection.size.positive?
+    return t('helpers.views.empty_collection', model: collection.model_name.human(count: 0).downcase) unless collection.size.positive?
```

Comment:
mm, maybe you can use something like:
`unless collection.any?`
or even
`if collection.empty?`